mico-core/src/main/java/io/github/ust/mico/core/resource/ServiceInterfaceResource.java

```diff
@@ -152,19 +183,17 @@
         }
 
         // Used to check whether the corresponding service exists
-        getServiceFromDatabase(shortName, version);
+        MicoService service = getServiceFromServiceBroker(shortName, version);
```

Comment:
Local variable `service` is never used. Consider to add a private helper method like `throwErrorIfServiceDoesNotExist` to make it more clear.